src/test-run/commands/index.js

```diff
@@ -491,6 +500,43 @@ export class PrepareBrowserManipulationCommand {
     }
 }
 
+export class ResizeWindowCommand extends Assignable {
+    constructor (obj) {
+        super(obj);
+
+        this.type   = TYPE.resizeWindow;
+        this.width  = 0;
+        this.height = 0;
+
+        this._assignFrom(obj, true);
+    }
+
+    _getAssignableProperties () {
+        return [
+            { name: 'width', type: positiveIntegerArgument, required: true },
+            { name: 'height', type: positiveIntegerArgument, required: true }
+        ];
+    }
+}
+
+export class ResizeWindowToFitDeviceCommand extends Assignable {
+    constructor (obj) {
+        super(obj);
+
+        this.type        = TYPE.resizeWindowToFitDevice;
+        this.device      = null;
+        this.orientation = null;
+        this._assignFrom(obj, true);
+    }
+
+    _getAssignableProperties () {
+        return [
+            { name: 'device', type: resizeWindowDeviceArgument, required: true },
+            { name: 'orientation', type: nonEmptyStringArgument }
```

Comment:
I think it makes sense to have options object with `{ portrait: Boolean }` property. If set to `false` - use `landscape` orientation (default).